libs/application/template-api-modules/src/lib/modules/shared/api/national-registry/EES.ts

```diff
@@ -0,0 +1,373 @@
+export const EES = [
```

Comment:
Using this for the EES citizenship check but unsure where the constant belongs. Any opinions ? @obmagnusson